site/content/contribute/server/developer-setup/windows-wsl.md

```diff
@@ -2,11 +2,26 @@ This is an unofficial guide. Community testing, feedback and improvements are we
 
 1. Install the Windows Subsystem for Linux: https://docs.microsoft.com/en-us/windows/wsl/install-win10
 
+      **Note:** Docker for Windows expects path to have the format `/c/foo/bar`, but WSL uses `/mnt/c/foo/bar` instead.
+      
+      Run `winver` and check which version of Windows you have. If you are using `1803` or higher, then tou need to create a file `/etc/wsl.conf` with the following content to make sure your drives are mounted at the root rather than inside `/mnt`:
```

Comment:
```suggestion
      Run `winver` and check which version of Windows you have. If you are using `1803` or higher, then you need to create a file `/etc/wsl.conf` with the following content to make sure your drives are mounted at the root rather than inside `/mnt`:
```